DER/DerivativesContracts/Forwards.rdf

```diff
@@ -260,113 +434,169 @@
-		<rdfs:domain rdf:resource="&fibo-der-der-fwd;ForwardCashSettlementCommitment"/>
-		<rdfs:range rdf:resource="&xsd;string"/>
-		<skos:definition xml:lang="en">The type of fee or additional payment, e.g. brokerage, upfront fee etc.</skos:definition>
+	<owl:ObjectProperty rdf:about="&fibo-der-der-fwd;lastDeliveryDate">
+		<rdfs:label xml:lang="en">last delivery date</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fbc-fi-fi;Future"/>
+		<rdfs:range rdf:resource="&fibo-fnd-dt-fd;Date"/>
+		<skos:definition xml:lang="en">The last date in the range of dates by which the underlying for a futures contract must be delivered in order for the terms of the contract to be fulfilled.</skos:definition>
+	</owl:ObjectProperty>
+	
+	<owl:ObjectProperty rdf:about="&fibo-der-der-fwd;lastNoticeDate">
+		<rdfs:label xml:lang="en">last notice date</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fbc-fi-fi;Future"/>
+		<rdfs:range rdf:resource="&fibo-fnd-dt-fd;Date"/>
+		<skos:definition xml:lang="en">The last date at which a delivery notice can be issued.</skos:definition>
+	</owl:ObjectProperty>
+	
+	<owl:DatatypeProperty rdf:about="&fibo-der-der-fwd;marginCall">
+		<rdfs:label xml:lang="en">has margin call</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-der-der-fwd;FuturesListing"/>
+		<rdfs:range rdf:resource="&xsd;boolean"/>
+		<skos:definition xml:lang="en">A broker&apos;s demand on an investor using margin to deposit additional money or securities so that the margin account is brought up to the minimum maintenance margin. This is sometimes called a &apos;fed call&apos; or &apos;maintenance call&apos;.</skos:definition>
 	</owl:DatatypeProperty>
 	
-	<owl:DatatypeProperty rdf:about="&fibo-der-der-fwd;paymentType">
-		<rdfs:label xml:lang="en">payment type</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-der-der-fwd;ForwardCashDeliveryCommitment"/>
-		<skos:definition xml:lang="en">&apos;A classification of the type of fee or additional payment, e.g. brokerage, upfront fee etc. FpML does not define domain values for this element.&apos;</skos:definition>
+	<owl:DatatypeProperty rdf:about="&fibo-der-der-fwd;marginCallApplicable">
+		<rdfs:label xml:lang="en">margin call applicable</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-der-der-fwd;StandardizedFuturesListingTerms"/>
+		<rdfs:range rdf:resource="&xsd;boolean"/>
+		<skos:definition xml:lang="en">Whether or not a Margin Call may be applicable to the listing.</skos:definition>
 	</owl:DatatypeProperty>
 	
+	<owl:ObjectProperty rdf:about="&fibo-der-der-fwd;marginProvision">
+		<rdfs:label xml:lang="en">has margin provision</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-der-der-fwd;FuturesListing"/>
+		<rdfs:range rdf:resource="&fibo-fnd-acc-cur;MonetaryAmount"/>
+		<skos:definition xml:lang="en">An amount necessary to cover the purchase of the underlying security since delivery is mandatory, be it physical or cash. Failing to provide a sufficient provision will result in a margin call.</skos:definition>
```

Comment:
The amount needed will vary over time, including after purchase, as the value changes.
See earlier comment on marginCall.